src/legacy/core_plugins/vis_type_table/public/components/table_vis_options.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useMemo } from 'react';
+import { get } from 'lodash';
+import { EuiPanel, EuiTitle, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { tabifyGetColumns } from 'ui/agg_response/tabify/_get_columns';
+import { VisOptionsProps } from 'ui/vis/editors/default';
+import {
+  NumberInputOption,
+  SwitchOption,
+  SelectOption,
+} from '../../../kbn_vislib_vis_types/public/components/common';
+import { TableVisParams } from '../types';
+import { totalAggregations, isAggConfigNumeric } from './utils';
+
+function TableOptions({
+  aggs,
+  aggsLabels,
+  stateParams,
+  setValidity,
+  setValue,
+}: VisOptionsProps<TableVisParams>) {
+  const percentageColumns = useMemo(
+    () => [
+      {
+        value: '',
+        text: i18n.translate('visTypeTable.params.defaultPercentageCol', {
+          defaultMessage: 'Don’t show',
+        }),
+      },
+      ...tabifyGetColumns(aggs.getResponseAggs(), true)
+        .filter(col => isAggConfigNumeric(get(col, 'aggConfig.type.name'), stateParams.dimensions))
+        .map(({ name }) => ({ value: name, text: name })),
+    ],
+    [aggs, aggsLabels, stateParams.percentageCol, stateParams.dimensions]
+  );
+
+  const isPerPageValid = stateParams.perPage === '' || stateParams.perPage > 0;
+
+  useEffect(() => {
+    setValidity(isPerPageValid);
+  }, [isPerPageValid]);
+
+  useEffect(() => {
+    if (
+      !percentageColumns.find(({ value }) => value === stateParams.percentageCol) &&
+      percentageColumns[0] &&
+      percentageColumns[0].value !== stateParams.percentageCol
+    ) {
+      setValue('percentageCol', percentageColumns[0].value);
+    }
+  }, [percentageColumns, stateParams.percentageCol]);
+
+  return (
+    <EuiPanel paddingSize="s">
+      <EuiTitle size="xs">
+        <h3>
+          <FormattedMessage
+            id="visTypeTable.params.showMetricsLabel.optionsTitle"
+            defaultMessage="Options"
```

Comment:
In Markdown the only panel is called "Settings" - not sure which one is better, do we even need one if there is just a single panel

Reply:
Let's better remove it for consistency, like such titles are skipped in `Controls`, `Coordinate map` and `Tag cloud`. Such titles are actually duplicates the tab name - `Options`